src/vivarium/framework/results/context.py

```diff
@@ -95,7 +95,7 @@ def add_observation(
         additional_stratifications: List[str],
         excluded_stratifications: List[str],
         when: str,
-        report: Callable[[Path, str, pd.DataFrame, str, str], None],
+        report: Callable[..., None],
```

Comment:
Is this really the right typing? It looks like it might be `Callable[[str, Any], None]`, but maybe it's just `Callable[[str, pd.DataFrame], None]`

Reply:
But since we're free to define whatever report method we want, then aren't the args unknown?